index.html

```diff
@@ -90,9 +90,19 @@
         </p>
         <pre class="idl">
           partial interface Navigator {
-            [SecureContext] Promise&lt;void&gt; share(ShareData data);
+            [SecureContext] Promise&lt;void&gt; share(optional ShareData data);
           };
         </pre>
+        <div class="note">
+          The <var>data</var> argument is marked as "optional", but really it
```

Comment:
Oh lol, use quotes next time. (I read it as "This really feel a bit superfluous".)